README.md

```diff
@@ -1 +1,70 @@
-## TODO PROJECT
+# TODO PROJECT
+
+## Local Development Setup
+
+This project uses [PNPM]([https://yarnpkg.com/getting-started](https://pnpm.io/motivation)) for package management.
+So, make sure to install PNPM to use the project.
+
+From your terminal:
+
+```sh
+pnpm dev
+```
+
+This starts your app in development mode, rebuilding assets on file changes.
+
+To build your app for production:
+
+```sh
+pnpm build
+```
+
+Then run the app in production mode:
+
+```sh
+pnpm start
+```
+
+## Project Structure
+
+We are using Next.js with App Router for this project. Next.js has a well-defined directory structure that must be used to make sure the app runs properly. Read more about Next.js [here](https://nextjs.org/docs/app/getting-started)
+
+### 'App' directory
+
+In the `Next.js` App Router, a page is a React Component exported from a `page.js`, `page.jsx`, `page.ts`, or `page.tsx` file in the `app` directory. Each page is associated with a route based on its file/directory name. Read more about `App` [here](https://nextjs.org/docs/app). An example is given below -
+
+#### Directory Structure
+
+```
+app
+|__ members
+|   |__ [id]
+|   |   |__ page.js
+|   |
+|   |__ page.js
+|
+|__ blogs
+|   |__ page.js
+|
+|__ page.js
+```
+
+#### Routes Created by Next.js
+
+```
+/
+/members
+/members/[id]
+/blogs
+```
```

Comment:
Can you please tell why is this part of our README? You can just share the next's docs links, those who want to learn can go there and learn.